react/features/toolbox/components/native/OverflowMenu.js

```diff
@@ -72,6 +73,7 @@ class OverflowMenu extends Component<Props> {
                 <RecordButton { ...buttonProps } />
                 <LiveStreamButton { ...buttonProps } />
                 <PictureInPictureButton { ...buttonProps } />
+                <TileViewButton { ...buttonProps } />
```

Comment:
Can you please put this as the previous to last? We have kept the PiP button as the last option for a while and IMHO it's nice to have it that way.